src/calculateRentalCost.js

```diff
@@ -5,6 +5,13 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
+  } else {
+    return days * 40;
+  }
```

Comment:
Consider using descriptive constant names for the values used in the calculations. This will make the code more readable and maintainable. For instance, use a constant for the daily rental rate, and constants for the long-term and mid-term discount thresholds and amounts, following the good example provided in the checklist.